libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.html

```diff
@@ -66,6 +66,12 @@
         (valueChange)="valueChange.emit($event)"
       ></gn-ui-form-field-overviews>
     </ng-container>
+    <ng-container *ngSwitchCase="'resourceCreated'">
+      <gn-ui-form-field-date-updated
```

Comment:
This does not have to be changed, but I was wondering if the name of this component is too specific as it serves date updated and date created now.

Reply:
I was also thinking of the same.
I guess I can change it to `gn-ui-form-field-date`